src/libraries/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.Cancellation.cs

```diff
@@ -301,7 +301,8 @@ public async Task CancelPendingRequest_DropsStalledConnectionAttempt()
         {
             if (UseVersion >= HttpVersion30)
             {
-                throw new SkipTestException("This test relies on ConnectCallback, and the scenario is not yet validated for HTTP/3.");
+                // This test relies on ConnectCallback, and the scenario is not yet validated for HTTP/3.
+                return;
```

Comment:
`SkipTestException` requires the test to be a `ConditionalFact` instead

Reply:
I remembered something like this but then I've seen this test which is decorated simply with `[Fact]`:

https://github.com/dotnet/runtime/blob/aafe4d465f97fa3a0ae61b18c2887cf84085d25d/src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.cs#L154-L178

I guess it works because of a side effect of `[SkipOnPlatform]`.